src/main/resources/data-repository-openapi.yaml

```diff
@@ -254,11 +323,46 @@ definitions:
 
   StudyModel:
     description: |
-      Complete description of a study.
+      Complete definition of a study.
     type: object
     properties:
       id:
         $ref: '#/definitions/UniqueIdModel'
+      name:
+        $ref: '#/definitions/DatasetNameModel'
```

Comment:
i'd like to change the name of "DatasetNameModel" to something a little more generic since it is used for both studies and datasets. It is slightly different from IdentifierNameModel. Maybe we should change the name of both of these so they indicate the difference. like "DBObjectName" instead of IdentifierNameModel and "ModelObjectName" instead of DatasetNameModel. I still get a little confused as to why we are using the "Model" suffix for these.

Reply:
We added the Model suffix so that when swagger generate turned into class names, they would be clearly labeled as coming from the REST API model and not one of our implementation classes. Having two classes named Dataset from different packages can be confusing. I'd be happy with a word other than Model, but would like some clue that the class is a generated REST API.

I notice that we are not applying the suffix consistently in this version. StudyRequest, StudySummary, DatasetRequest and DatasetSummary do not follow the pattern.